mozilla-release/toolkit/mozapps/installer/windows/nsis/common.nsh

```diff
@@ -5280,7 +5280,7 @@
 !ifmacrodef InitHashAppModelId
       ; setup the application model id registration value
       !ifdef AppName
-      ${InitHashAppModelId} "$INSTDIR" "Software\Mozilla\${AppName}\TaskBarIDs"
+      ${InitHashAppModelId} "$INSTDIR" "Software\${AppName}\TaskBarIDs"
```

Comment:
Good idea, but it is late, because we already launched product with path Software\CLIQZ